skfda/ml/regression/neighbors.py

```diff
@@ -319,15 +323,20 @@ def _init_multivariate_estimator(self, sklearn_metric):
             Sklearn Radius Neighbors estimator initialized.
 
         """
-        from sklearn.neighbors import (RadiusNeighborsRegressor as
-                                       _RadiusNeighborsRegressor)
-
         return _RadiusNeighborsRegressor(
             radius=self.radius, weights=self.weights,
             algorithm=self.algorithm, leaf_size=self.leaf_size,
             metric=sklearn_metric, metric_params=self.metric_params,
             n_jobs=self.n_jobs)
 
     def _query(self, X):
-        """Return distances and neighbors of given sample"""
+        """Return distances and neighbors of given sample.
```

Comment:
Start the sentence on the next line.

Reply:
Not sure what you're asking.

Reply:
This
```suggestion
        """
        Return distances and neighbors of given sample.
```

Reply:
Why do we need to do that?

Reply:
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
I usually prefer that all lines start on the same column, but it is only a preference.

Reply:
Okey, as it seems more standard to keep it in the same line and apparently that use is more spread in our code, I wouldn't change this.

Reply:
Ok.